packages/app/src/cli/models/app/loader.ts

```diff
@@ -467,7 +468,9 @@ export async function loadAppConfiguration(
   options: AppConfigurationLoaderConstructorArgs,
 ): Promise<AppConfigurationInterface> {
   const loader = new AppConfigurationLoader(options)
-  return loader.loaded()
+  const result = await loader.loaded()
+  await logMetadataFromAppLoadingProcess(result.configurationLoadResultMetadata)
```

Comment:
Isn't the [loaded function](https://github.com/Shopify/cli/pull/2775/files#diff-973c578c6a6700bec941fe428e1f488a4252f9fe356e5d002c6ce9d234137302R222) already adding the metadata? Do we need both? 🤔

Reply:
Correct!